### PR TITLE
Shotgun blast of small map fixes

### LINKED ISSUE #4308: Command teleporter has no emergency shutters
#### Brief description of the issue
Usually there are emergency shutters under doors. Command teleporter is missing one.

#### What you expected to happen
It to have one

#### What actually happened
It doesn't

#### Steps to reproduce
- Open door to command teleporter when its flooded with dangerous gas.
- Realize there was no shutter
- Cry

#### Code Revision
- db0f530d6e376aa765141f2b426c582edf45a4f9 
 
#### Anything else you may wish to add:
- http://prntscr.com/l1up4t

### LINKED ISSUE #4404: A stray airlock controller
#### Brief description of the issue
There is an airlock controller on asteroid 3 hangoutspot across from elevator that doesn't seem to be connected to anything.

#### What you expected to happen
No random buttons that do nothing.

#### What actually happened
https://prntscr.com/l6k4lr

#### Steps to reproduce
- Look at area across from elevator on asteroid 3

#### Code Revision
- 1fe04192c11e2ad7357512d5a873c44a4ea687d3

### PR DESCRIPTION
**Pull request by Mewchild**
- Closes #4308 (No Firelock on Command Teleporter)
- Adds various missing firelocks/emergency shutters through the map
- Fixes disconnected air vents in AI Core
- Removes Errant cobweb from solid wall in security
- Adds Stamps to the IA office
- Adds janitor entrance to south end of trash pit (allows them to throw away shit too large to fit in a chute)
- Shutters in cafe now cover tram-side windows
- Moved a trash pile behind the cargo mailing conveyor to a nearby section of maint that's not job restricted
- Removed a rock outcrop that stopped virology trash from being jettisoned into space
- Closes #4404 (Stray Airlock controller in Z7 hangout area)
- Removes Floating light in z7 janitor closet
- Flips security processing, to give the officer the side of the room with the intercom, light switch, etc
    - https://i.imgur.com/Vvby4bj.png

### CODE REVIEW

**Comment by Shophaune** [comment]
Something that maybe should have been brought up while you were asking for small issues... #4404

**Comment by Mewchild** [comment]
Notes for me, if I get to them before this is closed
- Xenobio lab is named Outpost xenobio lab.

File: maps/tether/tether-07-station3.dmm [hunk 1]
```diff
@@ -27301,10 +27339,17 @@
...
 /turf/simulated/floor/tiled,
 /area/security/eva)
-"Rm" = (
-/obj/effect/decal/cleanable/cobweb,
-/turf/simulated/wall/r_wall,
-/area/security/lobby)
+"Rs" = (
+/obj/machinery/button/windowtint{
+	step_x = 20;
+	step_y = 8;
```
Comment (Arokha) [inline maps/tether/tether-07-station3.dmm]: Nope.

Comment (Mewchild) [reply maps/tether/tether-07-station3.dmm]: Already fixing it. Caught it myself :P

**Comment by Mewchild** [comment]
Fuck it. Text file editing

**Comment by SpadesNeil** [comment]
Editing it manually is indeed the only way to correct it.

**Comment by Mewchild** [comment]
I won't be making any more map tweaks and edits. I've touched so many areas
in this already that I'm practically begging for a merge conflict.

**Pull request merged**